### ISSUE TITLE
Fix twitterfeed plugin, to work with new Twitter API

### ISSUE THREAD

**Issue by vdboor** [django-fluent/django-fluent-contents#18]
The `twitterfeed` plugin is no longer working properly, as the implementation we've used (http://tweet.seaofclouds.com/) no longer works. Quoting:

---

  Important note about Twitter's API changes
  Twitter has discontinued its unauthenticated v1.0 API, so this widget has stopped working as of 2013-06-11.

  Twitter wants your only option for on-page widgets to be their own Embedded Timelines, but there are workarounds which involve installing server-side code to support this widget. Please see this discussion for more details.

---

The twitterfeed would have to use Twitter's own widgets, or use a server-side solution instead.

**Comment by bashu** [comment]
@vdboor here it is https://github.com/edoburu/django-fluent-contents/pull/46

**Comment by bashu** [comment]
https://github.com/bashu/fluentcms-twitterfeed feedback? 

PS: I will upload it on PyPi tomorrow as usually =)

**Comment by sunjoomoon** [comment]
Another monkey go around will be to use 'rawhtml' plugin. The thing is that for new twitter timeline widget you need to login twitter and generate the one (including timeline) you need (getting something like ```data-widget-id="698355617065365504"```) to get it working.

1) login twitter and go to https://twitter.com/settings/widgets/new/user
2) create timeline and a code is generated for you to copy and paste. It looks something like this 

```
<a width="300" height="300" data-chrome="noheader nofooter noborders transparent noscrollbar" data-tweet-limit="3" class="twitter-timeline"  href="https://twitter.com/twitter" data-widget-id="698355617065365504">Tweets by @twitter</a>
<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
```
If you need customization, see https://dev.twitter.com/web/embedded-timelines#customization

**Comment by bashu** [comment]
@sunjoomoon just run:

```bash
pip install twitter-text  # instead of twitter-text-py
```
and use ``twitterfeed`` as before

**Comment by vdboor** [comment]
Thanks @bashu! I've released 2.0.3 to have the extra requires dependency fixed too.
